<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Yap</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f0f0f0;
}
QLabel {
    color: #333333;
}
QLineEdit {
    padding: 8px;
    border: 1px solid #cccccc;
    border-radius: 4px;
    background-color: white;
}
QPushButton {
    padding: 8px;
    border-radius: 4px;
    background-color: #4CAF50;
    color: white;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #45a049;
}
QPushButton#kayit_button {
    background-color: #2196F3;
}
QPushButton#kayit_button:hover {
    background-color: #1976D2;
}</string>
  </property>
  <widget class="QLabel" name="baslik_giris">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>160</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Giriş Yap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kullaniciadi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kullanıcı Adı:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="giris_kullaniciadi">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Kullanıcı adınızı giriniz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sifre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>100</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Şifre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="giris_sifre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Şifrenizi giriniz</string>
   </property>
  </widget>
  <widget class="QPushButton" name="giris_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Giriş Yap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_kayit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>350</y>
     <width>160</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hesabınız yok mu?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="kayit_button">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>300</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Kayıt Ol</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 